flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -73,7 +73,8 @@ public Integer getKey(Event value) throws Exception {
 					false,
 					keySelector,
 					IntSerializer.INSTANCE,
-					new NFAFactory()),
+					new NFAFactory(),
+					true),
```

Comment:
Should this be `false`? The parameter name is `migratingFromOldKeyedOperator`.

Same for the other instances in this file.